doc/files/package.json.md

```diff
@@ -219,7 +219,8 @@ Conversely, some files are always ignored:
 * `node_modules`
 * `config.gypi`
 * `*.orig`
-* `package-lock.json` (use shrinkwrap instead)
+
+`package-lock.json` is ignored by default, unless explicitly included (e.g. with `"*"`). Use shrinkwrap instead.
```

Comment:
O.o does this mean if package-lock is explicitly included, that `npm install` will respect `package-lock.json` files that are present inside installed dependencies?

Reply:
🤷‍♂️I only noticed that the file was included. I don't know about the consequences for packages.

It may very well be that the original docs were "correct" in terms of intended API, and that this inclusion behaviour is a bug

Reply:
We don't obey `package-lock.json` shipped with dependencies. Only `npm-shrinkwrap.json` is obeyed, so shipping these files is mostly an annoyance.

Reply:
Should that be expressed explicitly here in the docs?